e2e/rbd.go

```diff
@@ -1984,6 +1986,115 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create a encrypted block pvc and verify the image size", func() {
```

Comment:
Check snapshot process and size of created snapshot too in the same flow since we had a problem with restore?